--- Config_heavy.pl.orig	2014-07-05 01:59:15.000000000 +0900
+++ Config_heavy.pl	2014-07-05 02:02:41.000000000 +0900
@@ -1022,7 +1022,7 @@
 perl_static_inline='static __inline__'
 perladmin='root@e7913a0e40d0.(none)'
 perllibs='-lnsl -ldl -lm -lcrypt -lutil -lc'
-perlpath='/opt/perl/bin/perl'
+perlpath='.../perl'
 pg='pg'
 phostname='hostname'
 pidtype='pid_t'
@@ -1134,7 +1134,7 @@
 ssizetype='ssize_t'
 st_ino_sign='1'
 st_ino_size='8'
-startperl='#!/opt/perl/bin/perl'
+startperl='#!.../perl'
 startsh='#!/bin/sh'
 static_ext=' '
 stdchar='char'
@@ -1258,8 +1258,8 @@
 my $i = ord(8);
 foreach my $c (7,6,5,4,3,2,1) { $i <<= 8; $i |= ord($c); }
 our $byteorder = join('', unpack('aaaaaaaa', pack('L!', $i)));
-foreach my $what (qw(prefixexp archlibexp man1direxp man3direxp privlibexp scriptdirexp sitearchexp sitebinexp sitelibexp siteman1direxp siteman3direxp sitescriptexp siteprefixexp sitelib_stem installarchlib installman1dir installman3dir installprefixexp installprivlib installscript installsitearch installsitebin installsitelib installsiteman1dir installsiteman3dir installsitescript)) {
-    s/^($what=)(['"])(.*?)\2/$1 . $2 . relocate_inc($3) . $2/me;
+foreach my $what (qw(startperl perlpath prefixexp archlibexp man1direxp man3direxp privlibexp scriptdirexp sitearchexp sitebinexp sitelibexp siteman1direxp siteman3direxp sitescriptexp siteprefixexp sitelib_stem installarchlib installman1dir installman3dir installprefixexp installprivlib installscript installsitearch installsitebin installsitelib installsiteman1dir installsiteman3dir installsitescript)) {
+    s/^($what=)(['"])(#!)?(.*?)\2/$1 . $2 . ($3 || "") . relocate_inc($4) . $2/me;
 }
 s/(byteorder=)(['"]).*?\2/$1$2$Config::byteorder$2/m;
 
